workers/src/main/scala/workers/itc/ITCGraphRequests.scala

```diff
@@ -39,7 +39,7 @@ object ITCGraphRequests:
     targets:         NonEmptyList[ItcTarget],
     mode:            InstrumentRow,
     cache:           Cache[F],
-    callback:        GraphResponse => F[Unit]
+    callback:        ItcAsterismGraphResults => F[Unit]
```

Comment:
Do we need to invalidate the cache?

Reply:
This is used to send the results back to the client. Here it was just a type rename, the type was moved to `shared` into the `explore.model.itc` package.